src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/NotificationsServiceTests.cs

```diff
@@ -0,0 +1,222 @@
+                    },
+                    new()
+                    {
+                        Id = Guid.NewGuid(),
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                        Created = DateTime.UtcNow.AddDays(-1),
+                        NotifySubscribers = false,
+                    },
+                },
+            };
+
+            var release2 = new Release
+            {
+                Id = Guid.NewGuid(),
+                Publication = new Publication
+                {
+                    Id = Guid.NewGuid(),
+                    Title = "pub2 title",
+                    Slug = "pub2-slug",
+                },
+                ReleaseStatuses = new List<ReleaseStatus>
+                {
+                    new()
+                    {
+                        Id = Guid.NewGuid(),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                        Created = DateTime.UtcNow,
+                        NotifySubscribers = false,
+                    },
+                },
+            };
+
+            var release3 = new Release
+            {
+                Id = Guid.NewGuid(),
+                Publication = new Publication
+                {
+                    Id = Guid.NewGuid(),
+                    Title = "pub3 title",
+                    Slug = "pub3-slug",
+                },
+                ReleaseStatuses = new List<ReleaseStatus>
+                {
+                    new()
+                    {
+                        Id = Guid.NewGuid(),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                        Created = DateTime.UtcNow,
+                        NotifySubscribers = true,
+                    },
+                },
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddRangeAsync(release1, release2, release3);
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            var storageQueueService = new Mock<IStorageQueueService>(MockBehavior.Strict);
+            storageQueueService.Setup(mock => mock.AddMessages(
+                PublicationQueue,
+                It.IsAny<List<PublicationNotificationMessage>>()))
+                .Returns(Task.CompletedTask);
+
+            await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
+            {
+                var notificationsService = BuildNotificationsService(
+                    contentDbContext, storageQueueService.Object);
+
+                await notificationsService.NotifySubscribersIfApplicable(release1.Id, release2.Id, release3.Id);
+
+                storageQueueService.Verify(mock => mock.AddMessages(
+                    PublicationQueue,
+                It.Is<List<PublicationNotificationMessage>>(list =>
```

Comment:
Not 100% sure about this, but I think the test will still pass without `It.Is<List<PublicationNotificationMessage>>(list =>
                    list.SequenceEqual(` due to the equality methods that have been added on `PublicationNotificationMessage`

I'd be interested to know if there's any benefit to using `It.Is<List...` here